components/automate-ui/src/app/pages/integrations/list/integrations-list.component.html

```diff
@@ -67,3 +67,17 @@ <h2 class="display4 strong">Cloud Platform Providers</h2>
     </main>
   </div>
 </div>
+
+<chef-modal
+  id="delete-prompt"
+  [visible]="deletePromptVisible"
+  (closeModal)="resetModal()">
+  <h2 id="delete-prompt-title" slot="title">Delete Integration?</h2>
+  <div>
+    <p>Deleting this node manager will also delete all of its associated nodes</p>
+    <div class="options">
+      <chef-button label="confirm" primary caution (click)="handleDelete()">Delete</chef-button>
+      <chef-button label="cancel" tertiary (click)="resetModal()">Cancel</chef-button>
```

Comment:
@jonong1972 - Suggestions for wording on this Modal prompt for deleting a Node Manager (Integration)?

Reply:
```
Delete Integration?

This will remove all of its associated nodes and they will not be able to be scanned by scan jobs. To add them back you will need to create a new node integration.

Historical reported data on nodes will still be stored. 
```

@SEAjamieD Try that.